chain-vote/src/gang/p256k1.rs

```diff
@@ -57,12 +57,12 @@ impl GroupElement {
         }
     }
 
-    pub fn sum<'a, I>(mut i: I) -> Self
+    pub fn sum<'a, I>(i: I) -> Self
     where
         I: Iterator<Item = &'a Self>,
```

Comment:
An `IntoIterator` bound would make the API more flexible, but this change does not touch this, so could be done later.